.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,42 @@
+name: Run tests in Centos container
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  tests-on-centos:
+    strategy:
+      matrix:
+        version: ["c8s", "c9s"]
+    name: "Unittests on Centos ${{ matrix.version }}"
```

Comment:
nitpick: you could probably just omit `Centos` here, so it will end up being `Unittests on cXs`

Reply:
In that case, I think that using just a number for the version would be nicer. So `CentOS Stream 8` or `CentOS Stream 9`.

You can then use `quay.io/osbuild/osbuild-ci-c${{ matrix.version }}s` below...

Reply:
True, I have applied this change.